anda/system/cros-keyboard-map/cros-keyboard-map.spec

```diff
@@ -53,15 +53,15 @@ mkdir -p %buildroot/etc/systemd/system
 tee %buildroot/etc/systemd/system/cros-keyboard-map.service <<EOF
 [Unit]
 Description=Generate chromebook keyboard layout
-Before=keyd.service
 After=tmp.mount
 
 [Service]
 Type=oneshot
 ExecStart=/bin/bash /usr/bin/um-generate-cros-keymap
+ExecStartPost=+/usr/bin/systemctl restart keyd.service
 
 [Install]
-WantedBy=sysinit.target
+WantedBy=multi-user.target
 EOF
```

Comment:
1. wrong place to install this service to imo (`%_unitdir`)
2. don't like how this doesn't satisfy [-25](https://linear.app/fyra/issue/FYRA-25/move-in-tree-sources-into-seperate-repos)